util/joinpath.go

```diff
@@ -0,0 +1,306 @@
+// This file is Free Software under the MIT License
+// without warranty, see README.md and LICENSES/MIT.txt for details.
+//
+// SPDX-License-Identifier: MIT
+//
+// SPDX-FileCopyrightText: 2022 German Federal Office for Information Security (BSI) <https://www.bsi.bund.de>
+// Software-Engineering: 2022 Intevation GmbH <https://intevation.de>
+
+//go:build !go1.19
+
+// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
We need to properly attribute the file contents, this means the right REUSE headers, which includes adding the LICENSE to our LICENSES directory and a clear pointer to where we've gotten this file in particular.

Reply:
@s-l-teichmann can you add the precise link where you've gotten the file contents from.

Reply:
Should be fixed by now.